README.md

```diff
@@ -1,38 +1,110 @@
 # Ringing Room Bot
-A bot for Ringing Room that can fill in any set of bells to increase the scope of potential practices
+A bot for Ringing Room that can fill in any set of bells to increase the scope of potential practices.
 
-## Current features
-- Will anonymously ring any unassigned bells to plain courses of any method or any composition on CompLib (amongst other things).
-- Can detect and compenstate for changes in ringing speed made by the human ringers.
-- Adjustable sensitivity to speed changes (can be set to completely ignore other people), as well as adjustable pulloff speed (for when the bot is ringing both 1 and 2) and handstroke gap sizes.
-- Will respond to "Look to", "That's all", "Go", and "Stand" at the next handstroke after the call.
+This bot is designed to be a **'ninja helper with no ego'**.
+
+It will ring all unassigned bells perfectly whilst fitting into whatever you are doing, adjusting to
+your changes in rhythm.
+It will never try to take control of the ringing by giving instructions or powering on with a
+different rhythm to everyone else (unless you tell it to).
+However, by default it will adjust only to changes to overall rhythm - it will ignore individual
+bells holding up and ring in the 'right' place regardless.
+
+Only one ringer needs to run the command and it will ring with you after anyone calls `Look to` in
+Ringing Room.
+It understands `Go`, `Look to`, `That's all` and `Stand next` in Ringing Room, but currently can
+only ring plain courses of any given method (or touches from CompLib).
+
```

Comment:
I think the `MethodPlaceNotationGenerator` just assumes 4ths place bobs and singles, but it does understand them

Reply:
Oh does it - I didn't know that it did that...

Reply:
Updated :)